packages/aurora-ether/src/bridged-ether/sendToEthereum/index.ts

```diff
@@ -76,6 +76,8 @@ export interface TransferOptions {
   provider?: ethers.providers.Provider
   etherCustodianAddress?: string
   etherCustodianAbi?: string
+  etherCustodianProxyAddress?: string
+  etherCustodianProxyAbi?: string
```

Comment:
Yes, I think we can do it, but I don't want to because we need to explicitly separate etherCustodian and etherCustodianProxy in any case. The code will be clearer if the ABI is explicitly divided. This doesn't require much additional effort; we just need to extract one more file. For backward compatibility, we can duplicate the ABI and Address if the Proxy is not supported, and everything will work.